_posts/2021-07-07-cockpit-248.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Cockpit 248
+author: garrett
+date: '2021-07-07'
+tags: cockpit, machines, ostree
+slug: cockpit-248
+category: release
+summary: ''
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 248, Cockpit-Machines 247, and Cockpit-OSTree 187:
+
+
+## Metrics: Install missing packages
+
+[Cockpit 247](https://cockpit-project.org/blog/cockpit-247.html) introduced setting up a machine for Grafana metrics. However, this only worked if the `pcp` and `redis` packages were already installed. With this Cockpit version, the options will always work, and missing packages get installed as needed.
+
+![screenshot of install missing packages]({{ site.baseurl }}/images/248-metrics-install-missing-packages.png)
+
+![screenshot of install missing packages]({{ site.baseurl }}/images/248-metrics-install-missing-packages-2.png)
+
+This completes the story of [setting up PCP and Grafana metrics with Cockpit](https://cockpit-project.org/blog/pcp-grafana.html), have a look at that blog post for all the details.
+
+## PAM: Deprecate `pam_cockpit_cert` module
+
+The `pam_cockpit_cert` module is no longer required for client certificate authentication, and has been deprecated.  This release continues to ship a stub module which returns `PAM_IGNORE`, but this stub will be dropped in a future release.  If you modified  `/etc/pam.d/cockpit`, the package upgrade will not automatically update the file -- please remove the relevant entry manually in this case.
+
+## Machines: Bug fixes and improvements
+
+Cockpit-Machines has a new release with several bug fixes and minor improvements.
```

Comment:
I feel like we usually write "cockpit-machines"?

Reply:
For the name of the package, yes. For the name, we use Title Case.